blueprints/serverless/cloud-run-explore/README.md

```diff
@@ -0,0 +1,158 @@
+# Cloud Run Explore
+
+## Introduction
+
+This blueprint contains all the necessary Terraform modules to build and publicly expose a Cloud Run service in a variety of use cases.
+
+The content of this blueprint corresponds to the chapter '_My serverless "Hello, World! - Exploring Cloud Run_' of the __Serverless Networking Guide__ (to be released soon). This guide is an easy to follow introduction to Cloud Run, where a couple of friendly characters will guide you from the basics to more advanced topics with a very practical approach and in record time! The code here complements this learning and allows you to test the scenarios presented and your knowledge.
```

Comment:
When you release it, please also update the [REFERENCES.md](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/REFERENCES.md)